js/packages/web/src/components/AuctionCard/index.tsx

```diff
@@ -251,6 +253,9 @@ export const AuctionCard = ({
 
   //console.log("[--P]AuctionCard", tokenInfo, mintKey)
   const myPayingAccount = balance.accounts[0];
+  const instantSalePrice = useMemo(() =>
+    auctionView.auctionDataExtended?.info.instantSalePrice
```

Comment:
```suggestion
    auctionView.auctionDataExtended?.info.instantSalePrice.toNumber() || 0,
```

Reply:
can't do that. That variable is used in a transaction